### DISCUSSION TITLE
Linux/Auditbeat pipeline

### DISCUSSION THREAD

**Discussion by rado-van** [SigmaHQ/pySigma-plugin-directory#51] [Ask for Backends]
Hi,

will there be a pipeline for Linux / Auditbeat rules? Or which pipeline should be used for parsing Linux (and auditd) rules?

Thanks a lot.

**Comment by thomaspatzke** [comment]
Currently none is planned or known that someone implements one. I move this issue to the plugin directory, which is the central place for such requests.
